### DISCUSSION TITLE
Windows 11: Flameshot only captures primary screen.

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [flameshot-org/flameshot#3466] [General]
I have looked on various posts but cannot work out how to get Flameshot to work across all my monitors.

**Comment by madrats** [comment]
There's an issue with multiple screens - basically every time your screen configuration changes you have to restart Flameshot (this applies for Win10, but logically should apply to Win11 as well).
After that Flameshot should span all screens when launched.
